### ISSUE TITLE
Chat 1:1

### ISSUE THREAD

**Issue by datenbrei** [Human-Connection/Nitro-Web#43]
As a user I'd like to chat securely and confidentially with other people to share potentially sensitive information.

**Comment by appinteractive** [comment]
The questsion here is, which approach to use. We got some tests with Matrix / Riot as a service that could be used for chats. @datenbrei what were the experience with that service?

**Comment by sushidave** [comment]
Does this Chat 1:1 feature include the Private Message feature?

**Comment by appinteractive** [comment]
That’s what I understand under 1:1 yes.

**Comment by sushidave** [comment]
Oh, thanks @appinteractive. I was confused because on the illustration at https://human-connection.org/wp-content/uploads/2018/05/hc_baumload_A4_ENG_PRINT_160518.pdf the feature is called "Private Messaging".

**Comment by appinteractive** [comment]
Yeah, I really don’t know who still calls that “chat rooms” 😬 and yeah private messaging is way better then 1:1 😁

**Comment by roschaefer** [comment]
@datenbrei why is this part of the migration from `alpha`->`nitro`?
